src/misc/KeyManager.js

```diff
@@ -83,12 +81,34 @@ export function focusNext(dom: HTMLElement): boolean {
 	return true
 }
 
+type ShortcutOperation
+	= "register"
+	| "unregister"
+	| "registerModal"
+	| "unregisterModal"
+	| "registerDesktop"
+
+type ShortcutBatch = {
```

Comment:
Is this worth keeping if the original bug has been fixed? Will it help us avoid similar issues in the future, or should we just revert it?